### DISCUSSION TITLE
Unable to resolve "devlop" from "node_modules/react-markdown/lib/index.js"

### DISCUSSION THREAD

**Discussion by subraatakumar** [expo/expo#25492] [Archived: Questions]
I used `react-markdown` package in an expo app. For web it is working fine. But when i am trying to run it on android it shows error:

Android Bundling failed 111ms
Unable to resolve "devlop" from "node_modules/react-markdown/lib/index.js"

Please anyone help me

**Comment by MilanBarande** [comment]
Same issue here running my expo app on an iOs simulator with react-markdown 9.0.1, react-native 0.71.14 and expo 48.0.0
